.vscode/tasks.json

```diff
@@ -0,0 +1,26 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+        "label": "build",
+        "command": "./build.cmd",
+        "type": "shell",
+        "args": [
+        ],
+        "problemMatcher": "$msCompile",
+        "group": "build"
+    },
+    {
+        "label": "build skip analyzers",
```

Comment:
```suggestion
        "label": "build (skip analyzers)",
```

Reply:
Sorry. Saw this after. I'm fine if we want to change it to this form.